src/Layers/FeatureLayer/FeatureManager.js

```diff
@@ -240,7 +240,9 @@ export var FeatureManager = VirtualGrid.extend({
         // schedule adding features for the next animation frame
         Util.requestAnimFrame(Util.bind(function () {
           this.removeLayers(oldSnapshot);
-          this.addLayers(newSnapshot);
+          if (zoom < this.options.maxZoom && zoom > this.options.minZoom) {
```

Comment:
The `zoom` variable here is undefined. Is this supposed to be the current zoom level of the map (i.e. [`map.getZoom()`](http://leafletjs.com/reference-1.0.3.html#map-getzoom))?